### PR TITLE
Validator

### PR DESCRIPTION
**Pull request by maeste**

### CODE REVIEW

**Comment by jesperpedersen** [comment]
Straight port from the 1.2 branch as baseline.

Rename package.html -> package-info.java
The src/main/resources/validator.properties file is missing
The "validator" module needs to be included in the settings.gradle file before the "testsuite" module

TODOs are
* Separate configurations for the Ant, Maven and Gradle artifacts
* Standalone validator scripts
* Migrating the test suite to testsuite/

But I think those can wait until we got the initial deployment chain up and running

**Comment by maeste** [comment]
@jesperpedersen fixed w/ required changes. I've taken note of TODOs

**Comment by jesperpedersen** [comment]
@maeste There is a lot of /build/ related files in this

**Comment by maeste** [comment]
@jesperpedersen
​fixed and pushed. I've also fixed .gitignore for this

​

File: validator/src/main/resources/plugin.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<plugin>
```
Comment (jesperpedersen) [inline validator/src/main/resources/plugin.xml]: Remove this file for now

File: validator/src/main/resources/pom.xml [hunk 2]
```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jesperpedersen) [inline validator/src/main/resources/pom.xml]: Remove this file for now

File: validator/src/main/resources/validator.sh [hunk 3]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```
Comment (jesperpedersen) [inline validator/src/main/resources/validator.sh]: Remove this file for now

File: validator/src/main/resources/validator.bat [hunk 4]
```diff
@@ -0,0 +1 @@
+java -classpath ..\..\lib\ironjacamar-common-api.jar;..\..\lib\ironjacamar-common-impl.jar;..\..\lib\ironjacamar-common-spi.jar;..\..\lib\ironjacamar-spec-api.jar;..\..\lib\ironjacamar-validator-cli.jar;..\..\lib\ironjacamar-validator.jar;..\..\lib\jboss-common-core.jar;..\..\lib\jboss-logging.jar;..\..\lib\jandex.jar;..\..\lib\validation-api.jar;..\..\lib\hibernate-validator.jar org.jboss.jca.validator.cli.Main %*
```
Comment (jesperpedersen) [inline validator/src/main/resources/validator.bat]: Remove this file for now

File: validator/src/main/resources/validator.xml [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
```
Comment (jesperpedersen) [inline validator/src/main/resources/validator.xml]: Remove this file

**Comment by jesperpedersen** [comment]
Merged.

@maeste Please, start a new thread on ironjacamar-developers on the validator, thanks

**Pull request merged**